dagshub/logger.py

```diff
@@ -68,10 +68,10 @@ def epoch_milisec():
 
     def log_hyperparams(self, params: Dict[str, Any] = None, **kwargs):
         if self.should_log_hparams:
-            self.hparams.update(params or {})
-            self.hparams.update(kwargs)
-            if self.eager_logging:
-                self.save_hparams()
+          self.hparams.update(self.normalize_dictionary_values(params) or {})
```

Comment:
I think it makes more sense to check if params is None and this will automatically make `normalize_dictionary_values` never return None
```suggestion
          self.hparams.update(self.normalize_dictionary_values(params or {}))
```